test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/SmartRouterLoadBalancing.java

```diff
@@ -0,0 +1,92 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.integrationtests.scaling;
+
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.deployment.Instance;
+import org.kie.cloud.api.scenario.WorkbenchRuntimeSmartRouterKieServerDatabaseScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.common.provider.KieServerControllerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.maven.MavenDeployer;
+import org.kie.cloud.maven.constants.MavenConstants;
+import org.kie.server.api.model.KieContainerStatus;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.api.model.definition.ProcessDefinition;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.kie.server.integrationtests.controller.client.KieServerMgmtControllerClient;
+
+public class SmartRouterLoadBalancing extends AbstractCloudIntegrationTest<WorkbenchRuntimeSmartRouterKieServerDatabaseScenario> {
+
+    private KieServerMgmtControllerClient kieControllerClient;
+    private KieServicesClient kieServerClient;
+    private KieServicesClient kieServerClientRouter;
+
+    private static final int PROCESS_NUMBER = 100;
+    private static final String LOG_PROCESS_ID = "definition-project.logProcess";
+    private static final String LOG_MESSAGE = "Log process was started";
+
+    @Override protected WorkbenchRuntimeSmartRouterKieServerDatabaseScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory
+                .getWorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .build();
+    }
+
+    @Test
+    public void testRouterLoadBalancing() {
+        MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/definition-project-snapshot").getFile());
+
+        kieControllerClient = KieServerControllerClientProvider.getKieServerMgmtControllerClient(deploymentScenario.getWorkbenchRuntimeDeployment());
+        kieServerClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+
+        KieServerInfo serverInfo = kieServerClient.getServerInfo().getResult();
+        kieControllerClient.saveContainerSpec(serverInfo.getServerId(), serverInfo.getName(), CONTAINER_ID, CONTAINER_ALIAS, PROJECT_GROUP_ID, DEFINITION_PROJECT_SNAPSHOT_NAME, DEFINITION_PROJECT_SNAPSHOT_VERSION, KieContainerStatus.STARTED);
```

Comment:
Please reformat the code to avoid too-long lines. It's hard to read and when reviewing PRs on github it requires scrolling. I'd say that lines having 120+ characters (which is also the width of github's file preview) should be broken into multiple narrower lines.

Reply:
Thanks, it was reformated.